docs/src/faqs.md

```diff
@@ -28,33 +26,35 @@ observed(osys)
 Let's solve the system and see how to index the solution using our symbolic
 variables
 ```@example faq1
+u0 = [rn.A => 1.0, rn.B => 2.0, rn.C => 0.0]
+ps = [rn.k₊ => 1.0, rn.k₋ => 1.0]
```

Comment:
Without this rewriting, running thsi example takes crazy (infinite? I have a feeling that we have completed docs with it recently, so might finish, never bothered waiting that long in the retests though, but I have waited a good bit) long times, see this issue: https://github.com/SciML/Catalyst.jl/issues/886

Reply:
Instead of changing our docs can you make an MTK level example and open issue there so this can get fixed?

Reply:
I assume the same issue occurs with ODESystems and is related to the new symbolic indexing stuff.

Reply:
So right now I am trying to figure out what is going on enough that I can actually make an MTK level issue. Given that I have over 50 unfixed MTK issues right now, I don't feel very confident that this will be fixed anytime soon, unfortunately. It was not my intention to simply change the docs so that the issue can be ignored (hence I raised the Catalyst issue of a starter).

Reply:
(The change is only proposed, showing that some type of update is required here and what that could be, it is not meant as a final suggestion how the doc should be)

Reply:
Did this issue ever get resolved?

Reply:
not yet